### PR TITLE
Clean up and update spb-task-03

### PR DESCRIPTION
**Pull request by Freymaurer**

### CODE REVIEW

File: notebooks/english/spb-task-03-eng.ipynb [hunk 1]
```diff
@@ -4,23 +4,33 @@
...
+    "# Task 1: If-then-else"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Task 1.1\n",
     "\n",
     "Declare a function that takes a parameter `x` of type `float`. It shall return the value if `x` is greater than zero. If that is not the case, it shall return the value with the \n",
```
Comment (kMutagene) [inline notebooks/english/spb-task-03-eng.ipynb:30]: More concise Wording: which value should be returned? Better: "It shall return x if x is.." or "it shall return the value of x if x is ..."

File: notebooks/english/spb-task-03-eng.ipynb [hunk 2]
```diff
@@ -71,15 +80,14 @@
...
    "source": [
-    "### Task 1.3:\n",
+    "## Task 1.3\n",
     "\n",
-    "Declare a function that gets a year as parameter. If it is a leapyear, the result shall be `true` else `false`.\n",
-    "Hints:  \n",
-    "- If the year can be divided by four but not by 100, it is a leapyear. The year 2008 falls under this rule.  \n",
-    "- If the year can be divided by 100 but not by 400, it is no leapyear. 2100 will be no leapyear.  \n",
-    "- If the year can be divided by 400, it always is a leapyear. Because of this, the year 2000 was a leapyear.\n",
+    "Declare a function that gets a year as parameter. If it is a leap year, the result shall be `true` else `false`.\n",
```
Comment (kMutagene) [inline notebooks/english/spb-task-03-eng.ipynb:85]: More concise instruction: year as datetime or integer?

File: notebooks/english/spb-task-03-eng.ipynb [hunk 3]
```diff
@@ -132,9 +144,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 2.2:\n",
+    "## Task 2.2\n",
     "\n",
     "UnionCase definition `Month` is given:\n",
```
Comment (kMutagene) [inline notebooks/english/spb-task-03-eng.ipynb:149]: Move the union case into the code cell

File: notebooks/english/spb-task-03-eng.ipynb [hunk 4]
```diff
@@ -214,14 +225,17 @@
...
+    "# Task 3: Loops"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Task 3.1\n",
     "\n",
     "Declare a function that gets parameter `n` of type `int`.  \n",
```
Comment (kMutagene) [inline notebooks/english/spb-task-03-eng.ipynb:237]: Type: `a` parameter

File: notebooks/english/spb-task-03-eng.ipynb [hunk 5]
```diff
@@ -244,14 +258,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 3.2:\n",
+    "## Task 3.2\n",
     "\n",
     "Declare a function that gets parameter `n` of type `int`.  \n",
```
Comment (kMutagene) [inline notebooks/english/spb-task-03-eng.ipynb:263]: Same typo

File: notebooks/english/spb-task-03-eng.ipynb [hunk 6]
```diff
@@ -244,14 +258,15 @@
...
     "The function shall save all numbers from 1 to n in a list.  \n",
```
Comment (kMutagene) [inline notebooks/english/spb-task-03-eng.ipynb:264]: Concise instructions: values are not saved in a list. better: "the function shall return a list containing the values from 1 to n"

File: notebooks/english/spb-task-03-eng.ipynb [hunk 7]
```diff
@@ -274,13 +289,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 3.3:\n",
+    "## Task 3.3\n",
     "\n",
     "Declare a function that gets a parameter `n` of type `int` and a parameter `f` with signature `(int -> int)`.  \n",
-    "The function shall apply `f` onto all numbers from 1 to n and save them in a list.  \n",
-    "Hints: Sequence expressions, comprehensions  \n",
+    "The function should apply `f` to all integers from 1 to n and store them in a in a list. \n",
```
Comment (kMutagene) [inline notebooks/english/spb-task-03-eng.ipynb:295]: same as above

File: notebooks/german/spb-task-03-de.ipynb [hunk 8]
```diff
@@ -73,15 +80,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 1.3:\n",
+    "## Task 1.3\n",
     "\n",
     "Deklariere eine Funktion, die eine Jahreszahl als Parameter bekommt. Wenn es sich um ein Schaltjahr handelt, soll das Ergebnis `true` sein, andernfalls `false`.  \n",
```
Comment (kMutagene) [inline notebooks/german/spb-task-03-de.ipynb:85]: Here, hint if the date is an integer as well (see english notebook comment)

File: notebooks/german/spb-task-03-de.ipynb [hunk 9]
```diff
@@ -134,9 +144,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 2.2:\n",
+    "## Task 2.2\n",
     "\n",
     "Gegeben ist die UnionCase Definition `Monat`:\n",
+    "\n",
     "```fsharp\n",
```
Comment (kMutagene) [inline notebooks/german/spb-task-03-de.ipynb:151]: Move union case into code cell

File: notebooks/german/spb-task-03-de.ipynb [hunk 10]
```diff
@@ -247,14 +259,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 3.2:\n",
+    "## Task 3.2\n",
     "\n",
     "Deklariere eine Funktion, die einen Parameter `n` vom Typ `int` erhaelt.  \n",
-    "Die Funktion soll alle ganzen Zahlen von 1 bis n in einer Liste speichern.  \n",
+    "Die Funktion soll alle ganzen Zahlen von 1 bis n in einer Liste speichern.\n",
```
Comment (kMutagene) [inline notebooks/german/spb-task-03-de.ipynb:265]: see english notebook comment

File: notebooks/german/spb-task-03-de.ipynb [hunk 11]
```diff
@@ -277,13 +289,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 3.3:\n",
+    "## Task 3.3\n",
     "\n",
     "Deklariere eine Funktion, die einen Parameter `n` vom Typ `int` und einen Parameter `f` mit der Signatur `(int -> int)` erhaelt.  \n",
     "Die Funktion soll `f` auf alle ganzen Zahlen von 1 bis n anwenden und in einer in einer Liste speichern.  \n",
```
Comment (kMutagene) [inline notebooks/german/spb-task-03-de.ipynb:295]: see english notebook comment

File: notebooks/german/spb-task-03-de.ipynb [hunk 12]
```diff
@@ -343,10 +358,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 4.2:\n",
+    "## Task 4.2\n",
     "\n",
     "Modifiziere die folgende Funktion (bekannt aus der Vorlesung) so, dass sie -1 zurueckgibt, falls abzusehen ist, dass das Ergebnis groesser als 1000 wird.  \n",
```
Comment (kMutagene) [inline notebooks/german/spb-task-03-de.ipynb:363]: For these code blocks it is okay that they are in the markdown cell, because in contrast to the union case which should be used as-is, they need to be adapted.

**Review by kMutagene — CHANGES_REQUESTED** [review]

File: notebooks/english/spb-task-03-eng.ipynb (hunk 4 above)
Comment (Freymaurer) [reply notebooks/english/spb-task-03-eng.ipynb:237]: I don't really get what this means. Do you mean parameter of type `a`, parameter called `a` instead of `n`?

Comment (kMutagene) [reply notebooks/english/spb-task-03-eng.ipynb:237]: Declare a function that gets parameter -> Declare a function that gets *a* parameter (add article)

File: notebooks/english/spb-task-03-eng.ipynb [hunk 13]
```diff
@@ -291,8 +282,7 @@
    "source": [
     "## Task 3.3\n",
     "\n",
-    "Declare a function that gets a parameter `n` of type `int` and a parameter `f` with signature `(int -> int)`.  \n",
-    "The function should apply `f` to all integers from 1 to n and store them in a in a list. \n",
+    "Declare a function that gets a parameter `n` of type `int` and a parameter `f` with signature `(int -> int)`. The function should should apply `f` to all integers from 1 to ``n`` and then return a list of containing the results. \n",
```
Comment (kMutagene) [inline notebooks/english/spb-task-03-eng.ipynb:285]: same

File: notebooks/german/spb-task-03-de.ipynb [hunk 14]
```diff
@@ -27,7 +27,7 @@
    "source": [
     "## Task 1.1\n",
     "\n",
-    "Deklariere eine Funktion, die einen Parameter `x` vom Typ float erhaelt. Ist der Wert von `x` groesser Null, so soll die Funktion den Wert zurueckgeben, falls nicht soll das \n",
+    "Deklariere eine Funktion, die einen Parameter `x` vom Typ ``float`` erhaelt. Ist der Wert von `x` groesser Null, soll die Funktion `x` zurückgeben, falls nicht soll das \n",
```
Comment (kMutagene) [inline notebooks/german/spb-task-03-de.ipynb:30]: same

**Review by kMutagene — CHANGES_REQUESTED** [review]
